### PR TITLE
console - do not show pending users/orgs on the all users/org page

### PR DESCRIPTION
**Pull request by Gaetanbrl**
Link to #2456
Ask manual reviewers (no reviewers panel) @pierrejego @landryb

### CODE REVIEW

**Comment by fvanderbiest** [comment]
LGTM but requires a rebase. Thanks.

**Comment by Gaetanbrl** [comment]
> 
> 
> LGTM but requires a rebase.

Indeed, it's now necessary. Thanks for review.

File: console/src/main/webapp/manager/app/components/orgs/orgs.tpl.html [hunk 1]
```diff
@@ -34,7 +34,7 @@ <h3 class="roles-title">
       </thead>
 
       <tbody>
-        <tr dir-paginate="org in orgs.orgs | filter:orgs.q | itemsPerPage: orgs.itemsPerPage">
+        <tr dir-paginate="org in orgs.orgs | filter:orgs.q | filter: {pending: orgs.org=='pending'} | itemsPerPage: orgs.itemsPerPage">
```
Comment (RemiDesgrange) [inline console/src/main/webapp/manager/app/components/orgs/orgs.tpl.html]: maybe `===` ??

Comment (Gaetanbrl) [reply console/src/main/webapp/manager/app/components/orgs/orgs.tpl.html]: Please, could you more specific @RemiDesgrange about your request ?

**Comment by Gaetanbrl** [comment]
Rebase and requests applied @fvanderbiest. Thanks for reviews and tests.

**Pull request merged**